app/controllers/admin/orders_controller.rb

```diff
@@ -0,0 +1,121 @@
+class Admin::OrdersController < Admin::AdminController
+  include Pagy::Backend
+  before_action :find_order, only: [:show, :edit, :update, :cancel]
+
+  def index
+    @pagy, @orders = pagy(filtered_orders.order(sort_column => sort_direction))
+  end
+
+  def show
+    @order = Order.includes(order_items: :product).find(params[:id])
```

Comment:
```suggestion
    @product = @order.products
```
Như thế này cũng đc mà em. Em đã dùng before action để tìm được @order từ trước rồi mà